plugins/Files/js/fileElement.js

```diff
@@ -6,97 +6,115 @@
  */
 
 // Node modules
-const electron = require('electron');
-const clipboard = electron.clipboard;
 const $ = require('jquery');
 const tools = require('./uiTools');
 
-// Make file element with jquery
-function makeFileElement(f) {
-	var el = $(`
-		<div class='file' id='${f.name}'>
-			<div class='graphic'>
-				<i class='fa fa-${f.type}'></i>
-			</div>
-			<div class='name button'>${f.name}</div>
-			<div class='size'>${tools.formatByte(f.size)}</div>
-			<div class='time'></div>
-		</div>
-	`);
+// Update file element with jquery
+function updateFileElement(f, el) {
+	el = el || $('#' + f.hashedPath);
+	el.id = f.hashedPath;
 
 	// Set unavailable graphic if needed
 	if (!f.available) {
-		el.find('.fa.fa-file').removeClass('fa-file').addClass('fa-refresh fa-spin');
+		el.find('.fa.fa-file')
+			.removeClass('fa-file')
+			.addClass('fa-refresh fa-spin');
+	} else {
+		el.find('.fa.fa-refresh.fa-spin')
+			.removeClass('fa-refresh fa-spin')
+			.addClass('fa-file');
 	}
 
-	// Set time text
-	var timeText;
-	if (f.uploadProgress === 0) {
-		timeText = 'Processing...';
-	} else if (f.uploadProgress < 100) {
-		timeText = f.uploadProgress.toFixed(0) + '%'; 
+	// Set detail text
+	var detailText;
+	if (f.uploadprogress === 0) {
+		detailText = 'Processing...';
+	} else if (f.uploadprogress < 100) {
+		detailText = f.uploadprogress.toFixed(0) + '%'; 
+	} else if (f.renewing) {
+		detailText = 'Renewing';
```

Comment:
`Renewing` makes it sound like something is actively happening, when in reality the file is just set to be renewed as needed. I think in this case, nothing should be displayed at all.